MaterialLib/Component/Diffusion/Diffusion.h

```diff
@@ -0,0 +1,43 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file:   Diffusion.h
+ */
+
+#pragma once
+
+#include <ProcessLib/Parameter/Parameter.h>
+
+namespace MaterialLib
+{
+namespace Component
+{
+class Diffusion
```

Comment:
Is this class describing the molecular diffusion? Maybe a more explicit name would be  more suitable?